lib/src/compiler.dart

```diff
@@ -307,6 +307,8 @@ abstract class AbstractCompiler {
       filepath = 'dart/$filepath';
     } else if (uri.scheme == 'file') {
       filepath = path.relative(filepath, from: inputBaseDir);
+      // Play nice with paths not under inputBaseDir (which may be ''):
```

Comment:
This was a dirty hack needed when the input file was a relative file. Fixed by absolutizing the path in AbstractCompiler.stringToUri